content/chainguard/chainguard-images/working-with-images/scanners/grype-tutorial/index.md

```diff
@@ -0,0 +1,250 @@
+   ├── ✔ Packages                        [47 pac
+   │     └── ⠹ Linux kernel cataloger          
+   ├── ✔ File digests                    [659 fi
+   ├── ✔ File metadata                   [659 lo
+   └── ✔ Executables                     [144 ex
+ ✔ Scanned for vulnerabilities     [10 vulnerabi
+   ├── by severity: 0 critical, 1 high, 8 medium
+   └── by status:   7 fixed, 3 not-fixed, 0 igno
+NAME           INSTALLED   FIXED-IN    TYPE    VULNERABILITY        SEVERITY 
+busybox        1.36.1-r28  1.36.1-r29  apk     CVE-2023-42365       Medium    
+busybox        1.36.1-r28  1.36.1-r29  apk     CVE-2023-42364       Medium    
+busybox-binsh  1.36.1-r28  1.36.1-r29  apk     CVE-2023-42365       Medium    
+busybox-binsh  1.36.1-r28  1.36.1-r29  apk     CVE-2023-42364       Medium    
+pip            23.0.1      23.3        python  GHSA-mq26-g339-26xf  Medium    
+python         3.10.14                 binary  CVE-2023-36632       High      
+python         3.10.14                 binary  CVE-2023-27043       Medium    
+python         3.10.14                 binary  CVE-2024-4030        Unknown   
+ssl_client     1.36.1-r28  1.36.1-r29  apk     CVE-2023-42365       Medium    
+ssl_client     1.36.1-r28  1.36.1-r29  apk     CVE-2023-42364       Medium
+```
+
+### Interpreting the Summary
+
+In the initial portion of its results output, grype summarizes information on the scanned artifact and gives an overview of known vulnerabilities. In the case of a scanned image, the output includes the image digest, a unique hash of the image that can be used as an identifier. 
+
+Overview output includes the number of packages, files, and executables found in the artifact. Generally speaking, CVE are detected against packages, but the number of executables detected can also give you an idea of the attack surface of the scanned image or filesystem.
+
+Finally, this portion gives a count of the number of CVE detected by severity and fixed status. Severity categorization sorts CVE into four categories based on the Common Vulnerability Scoring System (CVSS).
+
+{{< details "What is CVSS??" >}}
+{{< blurb/cvss >}}
+{{< /details >}}
+
+In our output, we can see that we have 0 critical, 1 high, and 8 medium CVEs:
+
+```
+   ├── by severity: 0 critical, 1 high, 8 medium, 0 low, 0 negligible 
+```
+grype also counts the number of CVE by fixed status. If a CVE is marked as fixed, it can be resolved by updating to a newer version of the package. Our output suggests that 7 packages have been fixed and can be remediated with updates:
+
+```
+   └── by status:   7 fixed, 3 not-fixed, 0 ignored 
+```
+### Itemized CVEs
+
+In addition to the summary, grype provides an itemized list of CVE. By default, these are in table format, and list the pakage name, current version, the package type (such as apt, apk, binary), and severity. If the package is fixed, grype will also indicate the package version where the fix was introduced.
```

Comment:
```suggestion
In addition to the summary, Grype provides an itemized list of CVEs. By default, these are in table format, and list the package name, current version, the package type (such as `apt`, `apk`, `binary`), and the severity. If the package is fixed, Grype will also indicate the package version where the fix was introduced.
```

Reply:
This had pretty bad parallel structure so cleaned it up and moved the longer item to the end. Added the monospace.